### PR TITLE
Improve CLI

### LINKED ISSUE #22: Criar live reload ao alterar os arquivos do mock-server
Cenário: 
  Ao atualizar o db.json o mock-server poderia reiniciar o server para facilitar a manutenção.

### PR DESCRIPTION
**Pull request by LucasdeCastro**
This PR try resolve the issue #22 and improve the mock-server CLI.
Close #22

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     }
   ],
   "rules": {
-    "max-len": ["error", 100],
+    "max-len": ["error", 150],
```
Comment (rafaellucio) [inline .eslintrc.json]: Recommended return property length to 100.

File: bin/server.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const path = require('path')
+const mockServer = require('../app')
+
+const localPath = process.cwd()
+const { cacheOnly = false, mockServerConfigName = 'mock-server.conf.js' } = process.env
+
+const configFile = path.join(localPath, mockServerConfigName)
+
+try {
+  // eslint-disable-next-line
```
Comment (rafaellucio) [inline bin/server.js]: I don't understand why use eslint disable next line.

File: middlewares/request.js [hunk 3]
```diff
@@ -39,7 +39,9 @@ module.exports = () => async (req, res, next) => {
   } = req
 
   const parsedUrl = url.parse(originalUrl.replace(/^\/proxy/, ''))
-  const uri = `http://${URI_API}${parsedUrl.path}`
+  const uri = URI_API.includes('http')
```
Comment (rafaellucio) [inline middlewares/request.js]: In this case, you can have http and https, I recommended include https statement.

File: bin/cli.js [hunk 4]
```diff
@@ -35,138 +41,159 @@ const questions = [
...
+    env: {
+      cacheOnly: cache,
+      mockServerConfigName: fileName,
+    },
+    sourceDir: './bin/',
+  })
+}
+
+function infinitPrompt(props) {
+  // eslint-disable-next-line
```
Comment (guilhermegregio) [inline bin/cli.js]: it would be best to make it clear which rule is disabled

File: bin/cli.js [hunk 5]
```diff
@@ -35,138 +41,159 @@ const questions = [
...
+      process.stdout.write('\x1B[2J\x1B[0f')
+
+      switch (response.option) {
+        case 'r':
+          state.server.stop()
+          state.server = startServer(state.cacheOnly, state.silent, state.configFile)
+          break
+        case 'd':
+          state.cacheOnly = !state.cacheOnly
+          state.server.stop()
```
Comment (guilhermegregio) [inline bin/cli.js]: it would be better to change the state without stopping the server

**Pull request merged**